### ISSUE TITLE
Incorrect focus indicator on GitHub link at 'small' breakpoints 

### ISSUE THREAD

**Issue by ASM995** [mi6/ic-design-system#404]
## Summary of the bug

When the breakpoint is small enough that top navigation links are in the collapsed menu, the GitHub link focus indicator does not appear correctly. 

## 🪜 How to reproduce

Put any page at a small enough breakpoint that the top navigation has a collapsed menu and focus the GitHub link. 

## 🧐 Expected behaviour

Focus indicator should be around the whole button/text

## 📸 Screenshots or code

If you can, add screenshots to show the problem. If you need to show a code example, you can include a snippet or link to a repository with an example.

## 🖥 Desktop

- OS: [e.g. macOS]
- Browser: Chrome
- Browser version: [e.g. 22]
- Website version: [e.g. V1, V2]
  (As displayed on the site top navigation)

## 📱 Mobile

- Device: [e.g. iPhone13]
- Browser: [e.g. chrome, safari]
- Browser version: [e.g. 22]
- Website version: [e.g. V1, V2]
  (As displayed on the site top navigation)

## Additional info

Tell us anything else useful to help us fix or understand the problem.

**Comment by GCHQ-Developer-847** [comment]
Add `height: 100%` to the "button" class

<img src="https://user-images.githubusercontent.com/111882109/233353091-46df55a7-c731-4a5b-a29c-0c2a13f49d09.png" width="300">
